src/commands/generate/inspec_profile.ts

```diff
@@ -278,7 +278,7 @@ function getDISAReadmeContent(_xmlDoc: any): InspecReadme {
   readmeObj.profileShortName = benchmarkTitle.replace('Security Technical Implementation Guide', '').trim()
   readmeObj.profileTitle = benchmarkTitle
   readmeObj.profileVersion = `${stigVersion}.${stigRelease}.0`
-  readmeObj.inspecTitle = `${benchmarkTitle} :: Version ${stigVersion}, Release ${stigRelease} :: Benchmark Date: ${stigDate}`
+  readmeObj.inspecTitle = `${benchmarkTitle} :: Version ${stigVersion}, Release ${stigRelease} Benchmark Date: ${stigDate}`
```

Comment:
Will adding this here impact the GUI export in Heimdall? Have you tried to build Heimdall with this change in HDF converters and export a checklist through the GUI? If not, let me know and I can test that

Reply:
Oh I see that this is the README title. Thanks for pointing it out!